contracts/non-fungible-tokens/andromeda-crowdfund/src/contract.rs

```diff
@@ -507,10 +523,10 @@ fn purchase_tiers(
 
     // Update order history and sold amount for the tier
     set_tier_orders(deps.storage, full_orders)?;
-    current_cap = current_cap.checked_add(total_cost)?;
+    current_capital = current_capital.checked_add(total_cost)?;
 
     // Update current capital
-    set_current_cap(deps.storage, current_cap)?;
+    set_current_cap(deps.storage, current_capital)?;
```

Comment:
Could update that function's name to fully spell out "capital" while we're at it.